src/lib/logs/dynatrace_client.py

```diff
@@ -38,7 +37,7 @@ def send_logs(context: LogsContext, logs: List[LogProcessingJob], batch: str):
     # pylint: disable=R0912
     context.self_monitoring = aggregate_self_monitoring_metrics(LogSelfMonitoring(), [log.self_monitoring for log in logs])
     context.self_monitoring.sending_time_start = time.perf_counter()
-    log_ingest_url = urlparse(context.dynatrace_url + "/api/v2/logs/ingest").geturl()
+    log_ingest_url = urlparse(context.dynatrace_url.rstrip('/') + "/api/v2/logs/ingest").geturl()
```

Comment:
The change is ok. Consider changing the trailing '/' during the creation of the context varaible.

Reply:
Mm creating contexts is being done several times, I prefer to leave it here (or leave it here too, in addition, just in case).